src/CssToInlineStyles.php

```diff
@@ -101,7 +101,7 @@ public function getInlineStyles(\DOMElement $element)
     protected function createDomDocumentFromHtml($html)
     {
         $html = trim($html);
-        if (strstr('<?xml', $html) !== 0) {
+        if (strpos('<?xml', $html) !== 0) {
```

Comment:
@techi602 you can write tests using the public API which calls this one, i.e. trying to process some HTML containing the XML opening tag